roles/ckan/templates/kubernetes/ckan.yaml

```diff
@@ -86,6 +86,10 @@ spec:
               value: "/usr/lib/ckan/.minikubevenv"
             - name: XDG_CACHE_HOME
               value: "/usr/lib/ckan/.minikubevenv/cache"
+{% if ckan_googleanalytics_enable %}
+            - name: CKAN_GOOGLEANALYTICS_ENABLED
+              value: "1"
+{% endif %}
```

Comment:
```suggestion
            - name: CKAN_GOOGLEANALYTICS_ENABLED
              value: "{{ ckan_googleanalytics_enable }}"
```

Maybe this? (I find it much easier to follow this file if it's inline where possible!

Reply:
Also is nested inside `{% if fjelltopp_env_type == 'local' %}` whereas I assume we want it for all envs

Reply:
@ChasNelson1990 @cooper667 I have done as per your feedback.